esx_service/cli/vmdkops_admin.py

```diff
@@ -914,6 +913,13 @@ def generate_tenant_vm_ls_rows(vms):
 
 def tenant_vm_ls(args):
     """ Handle tenant vm ls command """
+
+    # Handling _DEFAULT tenant case separately to print info message
+    # instead of printing empty list
+    if (args.name == "_DEFAULT"):
+        print("All VMs that do not belong to any tenant belong to _DEFAULT tenant")
```

Comment:
the message is confusing IMO. Should be something "'__DEFAULT' tenant contains all VMs which were not added to other tenants"